src/w3c/l10n.js

```diff
@@ -24,6 +24,10 @@ const additions = {
     status_at_publication:
       "This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the <a href='https://www.w3.org/TR/'>W3C technical reports index</a> at https://www.w3.org/TR/.",
   },
+  es: {
+    status_at_publication:
+      "This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the <a href='https://www.w3.org/TR/'>W3C technical reports index</a> at https://www.w3.org/TR/.",
```

Comment:
Hmm, we need an actual Spanish translation. I’ll suggest one.

Reply:
Yeah sure, I don't know Spanish, and I don't consider Google Translate to be very accurate for technical content, so I left it in English.

Reply:
No problem. I know Spanish :) I need to check what’s been used in other document before.

Reply:
This is the translation, however could you add the missing anchor? See the original. 

> Esta sección describe el estado del presente documento al momento de su publicación. El presente documento puede ser remplazado por otros. Una lista de las publicaciones actuales del W3C y la última revisión del presente informe técnico puede hallarse en http://www.w3.org/TR/ el índice de informes técnicos del W3C

Reply:
Added the link above.

Reply:
Committed.